ray-operator/test/e2e/fail.py

```diff
@@ -0,0 +1,4 @@
+import sys
+
+print >> sys.stderr, "Something is seriously wrong."
```

Comment:
```suggestion
print("Something is seriously wrong.", file=sys.stderr)
```
I think we want to use Python 3 here? Though I guess it doesn't matter because we want it to fail anyway

Reply:
Agreed and updated.